core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3804,20 +3865,23 @@ public String toString() {
                   + "%s using the port defined in %s",
               Name.MASTER_RPC_ADDRESSES, Name.JOB_MASTER_RPC_PORT,
               Name.JOB_MASTER_EMBEDDED_JOURNAL_ADDRESSES, Name.JOB_MASTER_RPC_PORT))
-          .setScope(Scope.CLIENT)
+          .setScope(Scope.MASTER)
           .build();
   public static final PropertyKey JOB_MASTER_EMBEDDED_JOURNAL_ADDRESSES =
       new Builder(Name.JOB_MASTER_EMBEDDED_JOURNAL_ADDRESSES)
           .setDescription(String.format("A comma-separated list of journal addresses for all job "
               + "masters in the cluster. The format is 'hostname1:port1,hostname2:port2,...'. "
               + "Defaults to the journal addresses set for the Alluxio masters (%s), but with the "
               + "job master embedded journal port.", Name.MASTER_EMBEDDED_JOURNAL_ADDRESSES))
+          .setScope(Scope.MASTER)
```

Comment:
shouldn't this be Scope.ALL? Clients need that information to connect to the job master? Mainly for CLI use cases

Reply:
Changed to Scope.ALL, thanks